SU-SWS/stanford_basic
# updating-decanter: Needed to update decanter.

# READY FOR REVIEW

# Summary
- updates to get the new Decanter work

# Review By (Date)
- Oct 2

# Urgency
- High

# Steps to Test

1. Pull in this change
2. Add a media itme
3. Verify there is no empty link
4. Try undoing it and test with the link and content.
5. Verify there is a link around the content.

# Affected Projects or Products
- Decanter, SOE, Basic

# Associated Issues and/or People
- D8CORE-2529
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
